src/util/SpawnPed.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "core/frontend/Notifications.hpp"
+#include "game/backend/ScriptMgr.hpp"
+#include "game/rdr/Natives.hpp"
+
+
+namespace YimMenu
+{
+	extern int SpawnPed(std::string model_name, Player player, int playerPed, bool blockNewPedMovement, bool spawnDead, bool invincible, bool invisible, int scale);
```

Comment:
It would be easy to use if the latter params (which permeate relevance to the subject) had default values. This way we could get away with simplistic usage without having to define context-insensitive factors.